OpenRA.Game/ModData.cs

```diff
@@ -74,10 +74,7 @@ public ModData(string mod, bool useLoadScreen = false)
 			SpriteSequenceLoader = (ISpriteSequenceLoader)ctor.Invoke(new[] { this });
 			SpriteSequenceLoader.OnMissingSpriteError = s => Log.Write("debug", s);
 
-			// HACK: Mount only local folders so we have a half-working environment for the asset installer
-			GlobalFileSystem.UnmountAll();
```

Comment:
You may need to keep this line - check that things don't break when switching between mods.